CI failing when trying to import zarr

### What happened?

As of 08/04/2025, trying to import zarr for environments with Python 3.11 and 3.12 is producing an import error around 'numcodecs.blosc' in the CI.

https://github.com/openghg/openghg/actions/runs/14337742181/job/40189074167
```
ImportError while loading conftest '/home/runner/work/openghg/openghg/tests/conftest.py'.
...
    import zarr
/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/zarr/__init__.py:3: in <module>
    from zarr.convenience import (
/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/zarr/convenience.py:8: in <module>
    from zarr._storage.store import data_root, meta_root, assert_zarr_v3_api_available
/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/zarr/_storage/store.py:10: in <module>
    from zarr.meta import Metadata2, Metadata3
/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/zarr/meta.py:10: in <module>
    from zarr.util import json_dumps, json_loads
/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/zarr/util.py:33: in <module>
    from numcodecs.blosc import cbuffer_sizes, cbuffer_metainfo
E   ImportError: cannot import name 'cbuffer_sizes' from 'numcodecs.blosc' (/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/numcodecs/blosc.cpython-311-x86_64-linux-gnu.so)
Error: Process completed with exit code 4.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Drop `id` coordinate for GOSAT data to avoid merging errors [PR #1257](https://github.com/openghg/openghg/pull/1257)
 - Fixed bugs in ModelScenario for satellite data e.g. requiring max_level as argument [#PR 1261](https://github.com/openghg/openghg/pull/1261)
 - Fixed `get_*` functions if passed with `start_date` or `end_date` in format of ""dd:mm:yy T 00:00:0000" can still fetch the relevant data.[#PR 1273](https://github.com/openghg/openghg/issues/1273)
+- Fixed `numcodecs` version to be less than 0.16 to avoid ci runner failing while importing zarr.[# 1296](https://github.com/openghg/openghg/pull/1296)
 
 ## [0.13.0] - 2025-03-10
 

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,7 @@ ipywidgets
 matplotlib
 msgpack
 nbformat
-numcodecs
+numcodecs < 0.16
 nc-time-axis
 netcdf4
 numexpr

